src/Main.java

```diff
@@ -37,8 +37,10 @@ public static void main(String[] args) {
 		Runnable spawner = new Runnable() {
 			public void run() {
 				f.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
-				f.setTitle("Ein GUI");
-				f.setSize(800, 600);
+				f.setSize(1280, 720);
+				f.setResizable(false);
+				//f.setUndecorated(true);
```

Comment:
Wird das noch benötigt?

Reply:
Ja das ist wird noch benötigt, weil das den Fensterrahmen (also auch die Buttons Minimieren, Verkleinern/vergrößern und Schließen) entfernt, aber da das eine unsichere Sache ist bis der Exit-Button in jeden Screen ordnentlich implementiert ist. Sonst müsste man das Fenster schließen über Task-Manager oder sowas.

Reply:
Achso alles klar :) Kannst ja vllt kurz nen Kommentar da drüber schreiben, dass man bescheid weiß :)

Reply:
Wird gemacht.